frontend/src/axios-instance.ts

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: 'http://127.0.0.1:8000',
+});
+
+// Add an interceptor to include the access token in the request headers
+instance.interceptors.request.use((config) => {
+  const accessToken = localStorage.getItem('accessToken');
```

Comment:
I don't know if this is the way to do it properly tho. Might have to modify later.

Reply:
Yea I saw recommend to use cookie instead. Treat this as a temporary measure for now